pipeline-model-definition/pom.xml

```diff
@@ -108,6 +108,11 @@
       <version>${project.version}</version>
     </dependency>
     <dependency>
+      <groupId>org.jenkinsci.plugins</groupId>
+      <artifactId>pipeline-model-declarative-agent</artifactId>
+      <version>${project.version}</version>
```

Comment:
FWIW I do not recommend releasing reactors of plugins with synchronized versions. But if you really want to, you are doing it wrong here, since the `pipeline-model-declarative-agent` POM uses a fixed `version` rather than one inherited from an aggregator.